pages/linux/qm-sendkey.md

```diff
@@ -0,0 +1,12 @@
+# qm sendkey
+
+> Send QEMU monitor encoding key event to a specific virtual machine.
```

Comment:
```suggestion
> Send QEMU monitor encoding key event to a virtual machine.
```
I think we don't need to add `specific` keyword to the page description as it is already present in the command description. What do you think?

Reply:
Sounds good